src/utils/upload/putPartInS3.js

```diff
@@ -0,0 +1,44 @@
+import axios from 'axios';
+import fetchAPI from 'utils/http/fetchAPI';
+
+const MAX_RETRIES = 3;
+
+const putPartInS3 = async (
+  blob, uploadParams, partNumber, abortController, onUploadProgress, currentRetry = 0,
+) => {
+  try {
+    const signedUrl = await getSignedUrlForPart(uploadParams, partNumber);
```

Comment:
Move this into `FileUploader` and then this can be a normal `uploadToS3`

and rename `uploadFileToS3` to `multipartUploadFileToS3`